.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+### Issue ID
+
+e.g. fixes #123
+
+### Context/Purpose
+
+A sentence describing the issue. This should be readable by end users
+
+### Implementation details
+
+An overview of the approach taken to action the issue
+
+### PR Checklist
+
+- [ ] The title begins with the semantic-release format "type(Name): "
+- [ ] This is followed by a description of the task
+- [ ] I have added the issue number(s) to the title
+- [ ] The title matches the following format "type(Name): Description of the task [#123]"
```

Comment:
```suggestion
fixes #123

### Fixed
* A sentence describing each fix

### Added
* A sentence describing each addition

### Testing
1. Steps for testing

requires #123 

### PR Checklist
- [ ] The title begins with the semantic-release format "type(Name): "
- [ ] This is followed by a description of the task
- [ ] I have added the issue number(s) to the title
- [ ] The title matches the following format "type(Name): Description of the task [#123]"
```

Reply:
Not sure about the checklist part. I understand that their needs to be title enforcement for automation, but a fillable checklist seems a little too much. Any way to prepopulate it? Or at least shorten that text to something more deletable/informative.

Reply:
I've made some changes and put a lot of the descriptive elements in comments, so it should reduce the output noise. Can you take a look please?